packages/@okta/vuepress-site/docs/guides/social-login/main/apple/afterappidpinokta.md

```diff
@@ -0,0 +1,3 @@
+6. Copy both the **Authorize URL** and the **Redirect URI**, which ends in `/authorize/callback`.
+
+> **Note:** When you're setting up your IdP in Okta, there's a number of settings that allow you to finely control the social sign-in behavior. While the provider-specific instructions show one possible configuration, [Account Linking and JIT Provisioning](/docs/concepts/identity-providers/#account-linking-and-just-in-time-provisioning) discusses configuration options in more detail so that you can choose the right configuration for your use case.
```

Comment:
Have a question - I'm assuming that the reason this note isn't in the main shared index file is that some of the social login IdPs don't support Account Linking or JIT Provisioning? I noticed that not all of the individual files have this note. So, was just wondering if that was why?

Reply:
Yes, I'm assuming this too. This note was added to specific IdPs before I worked on these files, so I'm just preserving the info. I can revisit this issue at a later time to ask the SME specifically if accounting linking and JIT provisioning are supported in any new social IdPs.